spec/support/ansible_shared/automation_manager/credential.rb

```diff
@@ -1,8 +1,8 @@
 require 'ansible_tower_client'
 
-shared_examples_for "ansible credential" do |ansible_provider|
+shared_examples_for "ansible credential" do
   let(:finished_task) { FactoryGirl.create(:miq_task, :state => "Finished") }
-  let(:manager)       { FactoryGirl.create(ansible_provider, :with_authentication).managers.first }
+  let(:manager)       { ansible_manager }
```

Comment:
You don't need to do this.  You can drop this line if you rename [here](https://github.com/ManageIQ/manageiq/pull/14483/commits/31d99ebf5110050a608e66d6c6ca4ec673334545#diff-aee10777999bffa371c5d00941b7fd8bR4) and [here](https://github.com/ManageIQ/manageiq/pull/14483/commits/31d99ebf5110050a608e66d6c6ca4ec673334545#diff-8da487bcaedef7a5751de5c5796003d9R4).